tests/devapps/XForms/XForms.Android/XForms.Android.csproj

```diff
@@ -33,7 +33,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
     <AndroidLinkMode>None</AndroidLinkMode>
-    <AndroidSupportedAbis>armeabi;armeabi-v7a;x86</AndroidSupportedAbis>
```

Comment:
fyi: I had to revert all these changes locally to get the android app to even open.